go.mod

```diff
@@ -413,14 +413,14 @@ require (
 	golang.org/x/sys v0.0.0-20220209214540-3681064d5158
 )
 
-replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudexporter v0.46.0 => github.com/observiq/opentelemetry-collector-contrib/exporter/googlecloudexporter v0.0.0-20220304152956-bb36c08bd895
-
```

Comment:
Removing this breaks logs: 
```
2022/04/19 09:46:16 collector server run finished with error: cannot build exporters: error creating googlecloud exporter: failed to create logs exporter: telemetry type is not supported
```

When building, only the root modules replace directives take effect.

Reply:
Good catch. I didn't consider how that replace precedence would work.